src/test/java/api/requests/scenarios/HoldShelfExpirationDateTests.java

```diff
@@ -135,11 +183,7 @@ void requestWithShelfExpirationDateForSpExpiryInDaysWeeksMonths(
     assertThat("request status snapshot in storage is " + OPEN_AWAITING_PICKUP,
       storedRequest.getString("status"), is(OPEN_AWAITING_PICKUP));
 
-    ZonedDateTime expectedExpirationDate = atEndOfDay(interval.addTo(ClockUtil.getZonedDateTime(), amount));
-
-    assertThat("request hold shelf expiration date is " + amount + " " + interval.toString() + " in the future",
-      storedRequest.getString("holdShelfExpirationDate"),
-      isEquivalentTo(expectedExpirationDate));
+    assertHoldShelfExpirationDateBasedOnStrategy(storedRequest,servicePoint,amount,interval);
```

Comment:
Need spaces after each parameter in the method call so should be `storedRequest, servicePoint, ...`